packages/tasit-action/src/erc721/ERC721Full.test.js

```diff
@@ -132,29 +132,31 @@ describe("TasitAction.ERC721.ERC721Full", () => {
       await expectExactTokenBalances(erc721, [ana.address], [1]);
     });
 
-    // Non-deterministic
-    // Note: This test failed recently on CI
-    it.skip("should transfer an owned token", async () => {
+    it("should transfer an owned token", async () => {
```

Comment:
Oh cool, so this test became deterministic?

Reply:
Yes, until now, after a lot of executions, that wasn't failed.

Reply:
Great! Resolving this.